frontend/src/app/shared/components/dashboard/dashboard.component.html

```diff
@@ -5,7 +5,11 @@
     <div class="col-12 col-md-6">
       <bf-tweet-input *ngIf="showTweetInput" (submitTweet)="onSubmitTweet($event)"></bf-tweet-input>
       <ng-container *ngIf="tweets">
-        <bf-tweet-card *ngFor="let tweet of tweets" [tweet]="tweet"></bf-tweet-card>
+        <bf-tweet-card
+          *ngFor="let tweet of tweets"
+          [tweet]="tweet"
+          (deleteTweet)="onDeleteTweet(tweet.id)"
```

Comment:
EventEmitter of <bf-tweet-card> component passes the `tweetId` - take it from this Event instead. Otherwise update <bf-tweet-card> to to pass `tweetId`.